sdcm/tester.py

```diff
@@ -605,17 +605,6 @@ def get_cluster_aws(self, loader_info, db_info, monitor_info):
         # pylint: disable=too-many-locals,too-many-statements,too-many-branches
         regions = self.params.get('region_name').split()
 
-        def get_root_device_name(ami_image_id):
-            for region in regions:
-                ec2 = boto3.resource('ec2', region)
-                image = ec2.Image(ami_image_id)
-                try:
-                    if image.root_device_name:
-                        return image.root_device_name
-                except (TypeError, ClientError):
```

Comment:
Looks like it was the only place where `ClientError` was used. Need to remove unused import.

Reply:
thanks, removed.

Reply:
nice catch!